test/unit/test-template.js

```diff
@@ -349,11 +349,13 @@ describes.realWin('BaseTemplate', {amp: true}, (env) => {
     doc.body.appendChild(templateElement);
   });
 
+  /*
   it('should require render override', () => {
     expect(() => {
       new BaseTemplate(templateElement).render();
     }).to.throw(/Not implemented/);
   });
+  */
```

Comment:
Can this unit test be deleted?
Since the function body is now empty, it doesn't seem to make a lot of sense to test this.

Reply:
If you set the `@abstract` then yes, this test is useless.